etc/profile-a-l/balsa.profile

```diff
@@ -69,10 +69,11 @@ writable-run-user
 writable-var
 
 dbus-user filter
-dbus-user.own org.desktop.Balsa
 dbus-user.talk ca.desrt.dconf
-dbus-user.talk org.freedesktop.secrets
+dbus-user.own org.desktop.Balsa
 dbus-user.talk org.freedesktop.Notifications
+dbus-user.talk org.freedesktop.secrets
```

Comment:
Why reordering? All other profiles have own rules first, followed by the talk-rules.